src/pages/index.js

```diff
@@ -174,7 +174,9 @@ function Home() {
                   "button button--outline button--secondary button--lg button--hero get-started-use-case",
                   styles.getStarted
                 )}
-                to={useBaseUrl("docs/guides/orchestrate-microservices/")}
+                to={useBaseUrl(
```

Comment:
In past versions, I've deleted these links, because there was only one page of the three that existed. In this version, 2 of the 3 pages exist (but are in a different place), and I pointed the 3rd at a page that seems relatively similar.

Reply:
API Endpoints is misleading - it's the REST Connectors experience, so we are renaming it (#2444). 

We can delete these like you've handled it previously.

Reply:
Is this merge-blocking feedback?

Reply:
Yes

Reply:
removed in latest commit.

Reply:
Thank you!